### PR TITLE
made the duration tests more descriptive

### PR DESCRIPTION
**Pull request by jnation3406**

### CODE REVIEW

File: valhalla/userrequests/test/test.py [hunk 1]
```diff
@@ -127,17 +128,37 @@ def setUp(self):
             exposure_time=60, exposure_count=1, type='LAMPFLAT'
         )
 
+        self.sbig_fixed_overhead_per_exposure = 1
+        self.sbig_filter_change_time = 2
+        self.sbig_front_padding = 90
+        self.sbig_readout_time2 = 14.5
```
Comment (Fingel) [inline valhalla/userrequests/test/test.py]: I looked for sbig_readout_time1 but I couldn't find it.

File: valhalla/userrequests/test/test.py [hunk 2]
```diff
@@ -127,17 +128,37 @@ def setUp(self):
...
+
+        self.floyds_fixed_overhead_per_exposure = 0.5
+        self.floyds_filter_change_time = 0
+        self.floyds_front_padding = 240
+        self.floyds_readout_time1 = 25
```
Comment (Fingel) [inline valhalla/userrequests/test/test.py]: Similarly, there only appears to be one floyds readout time, not sure why there is a 1.

Comment (jnation3406) [inline valhalla/userrequests/test/test.py]: Those numbers refer to the binning for which that readout time is valid

**Comment by jnation3406** [comment]
I already added in a multi-spectrum test

File: valhalla/userrequests/test/test.py (hunk 2 above)
Comment (Fingel) [inline valhalla/userrequests/test/test.py]: Ah, I see. Cool.

**Comment by Fingel** [comment]
That would be test_floyds_multiple_spectrum_molecule_request_duration_with_acquire_on?

**Pull request merged**